pyiron_base/generic/hdfio.py

```diff
@@ -291,40 +291,50 @@ def copy_to(self, destination, file_name=None, maintain_name=True):
             FileHDFio: FileHDFio object pointing to a file which now contains the same content as file of the current
                        FileHDFio object.
         """
+        def _internal_copy(source, target, dest, h5_path):
+            if dest.h5_path[0] == "/":
+                dest_path = dest.h5_path[1:]
+            else:
+                dest_path = dest.h5_path
+            if maintain_name:
```

Comment:
Stylistically, I prefer a newline break between separate `if/elif/else` blocks to make them easier to read. If you like this or are neutral please change, if you hate this leave it how you have it.